docs/en/sql-reference/functions/other-functions.md

```diff
@@ -810,7 +810,8 @@ formatReadableTimeDelta(column[, maximum_unit, minimum_unit])
 
 **Arguments**
 
-- `column` — A column with a numeric time delta.
+- `column` — A column with a numeric time delta or an interval expression.
+  - Interval units of `MONTH` and greater are not supported as they don't map to a fixed time interval.
```

Comment:
Maybe this sounds a little more clear?
`- Interval units of `MONTH` and greater are not supported as they don't represent a fixed-sized interval in seconds.`

Reply:
Updated, thanks!